packages/common/test/items/follow.test.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  getEntityFollowersGroupId,
+  isUserFollowing,
+  followEntity,
+  unfollowEntity,
+} from "../../src/items/follow";
+import * as CoreModule from "../../src/core";
+import { IGroup, ISearchOptions, IUser } from "@esri/arcgis-rest-portal";
+import * as PortalModule from "@esri/arcgis-rest-portal";
```

Comment:
Super minor, but could we consolidate these imports here so we don't import twice? And just refer to `PortalModule.IGroup`, etc.?